proposals/0099-8-bit-types.md

```diff
@@ -0,0 +1,65 @@
+<!-- {% raw %} -->
+
+# 8-bit integer support
+
+* Proposal: [0099](0099-8-bit-types.md)
```

Comment:
We'll need to pick a proposal number when completing this.  At the same time, I think making the filename match the title might be appropriate here - since this proposal isn't talking about adding FP8 support, for example.

Something like `NNNN-8-bit-integers.md`.